distribution/scenes/templates/Textures_Materials/Metals/__index.htm

```diff
@@ -359,9 +359,8 @@
 <hr size="3">
 <table width="95%" align="center">
 <tr>
- <td><i><font size="-1">&copy; Friedrich A. Lohm&uuml;ller, 2010<br>
-
-   homepage:<a href="http://www.f-lohmueller.de"><nobr>http://www.f-lohmueller.de</nobr></font></i></a>
+ <td><i><font size="1">&copy; Persistence of Vision Ray Tracer, 2017<br>
```

Comment:
No, but the updated form is the copyright at the bottom in all but three of these html files from Friedrich. In looking at the pages nothing looked at all 'special' to me in a way which would make me think Friedrich wanted to maintain a copyright to just these 3 html pages.